po/common/common.pot

```diff
@@ -5731,4 +5731,20 @@ msgstr "Note: the Nutri-Score of teas and herbal teas corresponds to the product
 
 msgctxt "g_per_100g"
 msgid "%s g / 100 g"
-msgstr "%s g / 100 g"
\ No newline at end of file
+msgstr "%s g / 100 g"
+
+msgctxt "donation_title"
+msgid "Important: we need your support!"
+msgstr "" 
+
+msgctxt "donation_body"
+msgid "Open Food Facts is a collaborative project built by tens of thousands of volunteers and managed by a non-profit organization with 3 employees."
+msgstr "" 
+
+msgctxt "donation_why"
+msgid "We need your donations to <a href=\"[% link %]?utm_source=off&utm_medium=web&utm_campaign=donate-2020&utm_term=en-text\">fund the Open Food Facts 2021 budget</a> and to continue to develop the project. <strong>Thank you! ❤️</strong>"
```

Comment:
The template generation is not recursive, so [% link %] will not be interpreted. The easiest is probably to remove the link from the text, and keep it in the Donate button.